manifest/v1alpha/slo/composite.go

```diff
@@ -0,0 +1,40 @@
+package slo
+
+// CompositeSpec represents a composite of SLOs and Composite SLOs.
+type CompositeSpec struct {
+	MaxDelay   string `json:"maxDelay"`
+	Components `json:"components"`
```

Comment:
why is it embedded if we still pass it as `components.objectives` instead of  just `objectives` in JSON?

Reply:
This is to prepare schema for a feature that might be one day implemented where user doesn't set an explicit set  of objectives but rather sets a filter that will dynamically evaluate for such set. Such filter would be under `components` field but mutually exclusive with `objectives` list.